src/LoginLink.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import cookie from 'react-cookie';
-import { browserHistory, Link } from 'react-router';
+import { browserHistory } from 'react-router';
+
+const client_id = (process.env.REACT_APP_GITHUB_CLIENT_ID);
+const authURL = `https://github.com/login/oauth/authorize?scope=user:email%20read:org&client_id=${client_id}`;
```

Comment:
We'll be hitting the same endpoint regardless of which environment is being used, so I don't think that's needed